### PR TITLE
Bio

### PR DESCRIPTION
**Pull request by dsanchez185**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/bio/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <col align="left">
+      <col align="right">
+      <tr>
+        <td class="header">Diana Sanchez</td>
+        <td>
+          <ul class="navigationarea">
+            <li class="listarea"><a href="https://www.linkedin.com/in/diana-sanchez185/">LinkedIn</a> &nbsp;|</li>
+            <li class="listarea"><a href="https://github.com/dsanchez185">Github</a> &nbsp;|</li>
+            <li class="listarea"><a href="tel:2104124915">Phone</a> &nbsp;|</li>
+            <li class="listarea"><a href="malito:dsanchez185@gmail.com">Email</a></li>
```
Comment (shelbyinloes) [inline 02week/bio/index.html]: spelling error in "mailto:"

**Review by shelbyinloes — COMMENTED** [review]
Diana, this looks awesome!! It all works well. There is one change to make and its a simple spelling error, I made a comment on it.

**Comment by shelbyinloes** [comment]
Diana, once you fix the spelling error we can merge this. Looks amazing! Nice work.

**Comment by dsanchez185** [comment]
Okay, I have made the edit and pushed the branch. Thanks!!

**Pull request merged**